public/colors.css

```diff
@@ -27,6 +27,7 @@
   --color-lightgray-tint-10: #e9edf1;
   --color-lightgray: #ced8e1;
   --color-lightgray-shade-10: #b3c3d1;
+  --color-light-shadow-gray: rgba(51, 51, 51, 0.08);
```

Comment:
`colors.css` is auto-generated and will be overwritten next time someone runs `yarn generate:colors`.

Let's put this CSS variable in `public/global.css` for now.
I also think `--color-lightgray-transparency-08` would better fit our current naming scheme.